packages/components/src/constants.ts

```diff
@@ -210,3 +210,10 @@ export const VISUALIZATION_REPORTS = Set([
     DataViewInfoTypes.XYScatterPlot,
     DataViewInfoTypes.XYSeriesLinePlot,
 ]);
+
+export enum IMPORT_DATA_FORM_TYPES {
+    GRID = 1,
```

Comment:
Is there a reason why GRID has a 1 and the others don't have numeric values set? Just curious.

Reply:
This is just the starting point for the enum.  By default it starts at 0, but since that is falsy, it makes the check `if (changeType)` problematic.  That could be changed to `if (changeType == undefined)` of course, but this seemed better.

Reply:
Cool, good to know. Thanks.

Reply:
This seems like the kind of info that's nice to capture in a comment in the code, not just in the PR